data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -501,6 +502,34 @@ def add_HARMONIZED_MEDIA_TYPE_column_using_fixed_str_value(
                  comp_harm_constants.MEDIA_TYPE_COLUMN,
                  fixed_str_value)
 
+    def add_NOT_AVAILABLE_for_empty_values_in_HARMONIZED_ADVERTISER_column(
```

Comment:
Why don't we combine these two functions? They are doing the SAME thing and you are repeating the code. 

So instead of having two functions, create just one function with a name like, "replace_empty_values_with_NOT_AVAILABLE(self, df, column_name):"

Reply:
Renamed.

Reply:
And the function snow is just one, replace_empty_values_with_NOT_AVAILABLE.

Reply:
Done.